canonical-web-and-design/ubuntu.com
# UA renewal info markup

# Done

- Updated the layout of /advantage based on the feedback received below. When a renewal is due to expire soon, there is now information to that effect in the table indicating how many days are left. This PR doesn't provide a way for the user to do anything about it, that will come with the [Stripe integration work](https://github.com/canonical-web-and-design/ubuntu.com/issues/6202).

# QA

**QAing this is dependent on you having some subscriptions on the staging contracts service**

- Check out this branch, or visit the demo link
- Visit /advantage
- Log out, if you aren't already logged out
- See that the page looks good
- Click the sign in button at the top of the page
- If you have subscriptions associated with your account that have "effective to" dates, see that there is warning information indicating how many days until they expire.
- See that the page looks good

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2481

## Screenshot

![FireShot Capture 002 - Ubuntu Advantage for Infrastructure - Ubuntu_ - ubuntu-com-canonical-web-and-design-pr-7042 run demo haus](https://user-images.githubusercontent.com/2376968/78347316-33a2af80-7598-11ea-9eff-8b8d62d53fd5.png)
